packages/api-fetch/tsconfig.json

```diff
@@ -8,11 +8,13 @@
 		{ "path": "../url" }
 	],
 	"include": [
+		"src/middlewares/media-upload.js",
 		"src/middlewares/namespace-endpoint.js",
 		"src/middlewares/nonce.js",
 		"src/middlewares/preloading.js",
 		"src/middlewares/root-url.js",
 		"src/middlewares/user-locale.js",
+		"src/utils/**/*",
```

Comment:
Do we really need the `**/` bit? Or that's here just in case we eventually end up with a hundred of those utils so we need directories to group them? 😉

Reply:
Yes, if `utils` grows this will make sure that everyone continues to include strict typings on the new files 😁

Reply:
Well, the thing is it may never grow that big. I'd say YAGNI, but also feel free to leave it as-is, since it doesn't hurt anyone that way. Your call 😉

Reply:
Yeah it doesn't hurt anyone and will hopefully go away soon anyway once the package is fully typed. When the package is fully typed it'll be transformed to `src/**/*` to catch everything anyway.